flask_of_oil/oauth_filter.py

```diff
@@ -141,6 +147,7 @@ def filter(self, scopes=None):
             abort(make_response("Forbidden", 403))
 
         # Set the user info in a context global variable
-        g.user = validated_token['subject']
+        g.user = validated_token['sub']
+        request.foil_claims = validated_token
```

Comment:
also contains `sub`. maybe remove `sub` on line 150 as you assign it to `g.user`.

Reply:
Or not assign it to g.user? I don't see a reason why that should stay there